client/src/pages/SurveyDetail/index.jsx

```diff
@@ -287,20 +282,20 @@ const SurveyDetail = ({ match }) => {
           {status === 'draft' && (
             <Typography>To edit recipient list, select Edit Survey</Typography>
           )}
+
           <EmployeeCompletionTable />
+          {employeeData && <SlackModal />}
         </Box>
       )}
       {successfulPublish !== undefined && <SnackbarPublish />}
       {successfulClose !== undefined && <SnackbarClose />}
```

Comment:
Lines 290 and 291 should just be,
`{successfulClose && <SnackbarClose />}`

Reply:
nice, this wasn't part of my code but have changed it now!